code/modules/power/tesla/energy_ball.dm

```diff
@@ -21,13 +21,22 @@
 	var/energy_to_lower = -20
 	var/max_balls = 10
 	var/zap_range = 7
+	var/hypercharged = FALSE //if true, will not lose energy and tesla zap will dust you
 
-/obj/singularity/energy_ball/Initialize(mapload, starting_energy = 50, is_miniball = FALSE)
+/obj/singularity/energy_ball/Initialize(mapload, starting_energy = energy, is_miniball = FALSE)
```

Comment:
does ``starting_energy`` do anything here?

Reply:
yes, 
`/obj/singularity/Initialize(mapload, starting_energy = 50)
	//CARN: admin-alert for chuckle-fuckery.
	admin_investigate_setup()

	src.energy = starting_energy`

Reply:
i have to change that, or energy ball will be init with 50 instead other numbers

Reply:
is it just not used in this file?

![image](https://github.com/yogstation13/Yogstation/assets/53777086/3b6bbf1c-8c79-4e5b-a77d-feca88dc0a0f)

Reply:
in the singularity

Reply:
singularity.dm line 51 i think